chanzuckerberg/czgenepi
# Maya/download modal refactor

### Summary
- **What:** Create a component for download modal options. 
- **Why:** It was super hard to parse style tags vs relevant content before. Also there were a couple styling bugs.

### Testing
1. Select a sample that failed genome recovery
1. Check out the download modal
2. Top option should be disabled and have tooltip on hover

### Demos
No visual changes.

### Checklist
